docs/guide/deploying_contracts.rst

```diff
@@ -77,8 +77,8 @@ Declaring Cairo1 contracts
 
 | Starknet 0.11 introduced the ability to declare contracts written in Cairo1!
```

Comment:
As discussed offline, this should be changed. However, since the current logic at some points in the code allows only Cairo version 0 or 1, we will not update the documentation until it is changed.